server/server.go

```diff
@@ -1583,6 +1584,18 @@ func PrintServerAndExit() {
 	os.Exit(0)
 }
 
+// PrintBuildinfoAndExit will print out https://pkg.go.dev/runtime/debug#BuildInfo and exit.
+func PrintBuildinfoAndExit() {
```

Comment:
should this be exported? Doesn't seem like something that would be useful when embedded or have use as a callable API really

Reply:
I was going to make the same comment, grant it the `PrintServerAndExit` function is also exported. I am generally in favor of exporting less..